conda/activate.py

```diff
@@ -943,7 +954,7 @@ def __init__(self, arguments=None):
         super(XonshActivator, self).__init__(arguments)
 
     def _hook_preamble(self):
-        return '$CONDA_EXE = "%s"' % context.conda_exe
+        return '$CONDA_EXE = "%s"' % self.path_conversion(context.conda_exe)
```

Comment:
Ahh.  `self.path_conversion` goes to the parent class since `XonshActivator.path_conversion()` is a static method.

Reply:
Or maybe not. More likely just because the ``__init__`` overrides the ``path_conversion`` name.